pkg/apis/openliberty/v1beta1/openlibertyapplication_types.go

```diff
@@ -42,10 +43,15 @@ type OpenLibertyApplicationSpec struct {
 	CreateKnativeService *bool                             `json:"createKnativeService,omitempty"`
 	Monitoring           *OpenLibertyApplicationMonitoring `json:"monitoring,omitempty"`
 	CreateAppDefinition  *bool                             `json:"createAppDefinition,omitempty"`
+	ApplicationName      string                            `json:"applicationName,omitempty"`
 	// +listType=map
 	// +listMapKey=name
-	InitContainers []corev1.Container                    `json:"initContainers,omitempty"`
-	Serviceability *OpenLibertyApplicationServiceability `json:"serviceability,omitempty"`
+	InitContainers []corev1.Container `json:"initContainers,omitempty"`
```

Comment:
(not directly related to this line)
@arturdzm do you know why the following is different from Runtime Component operator:
```
	// +listType=map		// +listType=map
	// +listMapKey=name		// +listMapKey=name
	VolumeMounts        []corev1.VolumeMount          `json:"volumeMounts,omitempty"`
```
vs
```
	// +listType=atomic
	VolumeMounts        []corev1.VolumeMount         `json:"volumeMounts,omitempty"`
```